webapp/src/main/webapp/src/styles.less

```diff
@@ -139,6 +139,7 @@ footer {
 .my-drop-zone {
   border: dotted 3px lightgray;
   border-radius: 5px;
+  height: 118px; /* Required for drag & drop to work in IE11 */
```

Comment:
Would it still work with min-height?
I'm thinking if there is another more verbose message (like if we later switch to Vietnamese) occupying the same space we wouldn't want overflow

Reply:
There's a lot of room to grow...

![image](https://user-images.githubusercontent.com/5771116/30657953-0a1a267c-9dee-11e7-9bd8-8971e3193bdf.png)

The drag and drop area might be restricted to the min-height area in IE11, would need to test.